libkbfs/folder_branch_ops.go

```diff
@@ -3753,8 +3753,12 @@ func (fbo *folderBranchOps) startSyncLocked(ctx context.Context,
 		// perfectly accurate (but at the same time, we'd then have to
 		// fix up the intentional panic in the background flusher to
 		// be more tolerant of long-lived dirty, removed files).
-		return false, true, nil, nil, nil, fileSyncState{}, nil,
-			fbo.blocks.ClearCacheInfo(lState, file)
+		err := fbo.blocks.ClearCacheInfo(lState, file)
+		if err != nil {
+			return false, false, nil, nil, nil, fileSyncState{}, nil, nil
+		}
+		fbo.status.rmDirtyNode(node)
+		return false, true, nil, nil, nil, fileSyncState{}, nil, nil
```

Comment:
If it's `false`, the node will be removed by the caller redundantly.

I was thinking about getting rid of that second return param entirely, not sure if it's still meaningful, but I didn't want to change too much in this PR.